src/Symfony/Component/Security/Core/Exception/AuthenticationExpiredException.php

```diff
@@ -12,7 +12,7 @@
 namespace Symfony\Component\Security\Core\Exception;
 
 /**
- * AuthenticationServiceException is thrown when an authenticated token becomes un-authentcated between requests.
+ * AuthenticationServiceException is thrown when an authenticated token becomes un-authenticated between requests.
```

Comment:
Perhaps all sub classes of `AuthenticationException.php` just need to use `{@inheritdoc}` :rofl: 

I guess this exception describes as `thrown if authenticated token is expired` which is already implied by name.. hence i think @inheritdoc seems fine.